Tests/com/drew/lang/SequentialAccessTestBase.java

```diff
@@ -241,24 +241,39 @@ public void testGetNullTerminatedString() throws IOException
     }
 
     @Test
-    public void testGetNullTerminatedStringCursorPositionTest() throws IOException
-    {
+    public void testGetNullTerminatedStringCursorPositionTest() throws IOException {
```

Comment:
Please avoid reformatting code in PRs that also include functional changes. Formatting changes are appreciated when they change code towards the predominant style in the repo. We don't need the `getNullTerminatedStringAndSkipToNextPosition` method, nor its unit test. The `toString` calls below don't seem to do anything, and the formatting changes are unrelated to the PR. I think this file can be exluded from the PR altogether.